index.ios.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+import type { Region } from './models/Location';
+import type { Callback } from './models/Callback';
+import debounce from './utils/debounce';
+
+const { RNReverseGeocode } = NativeModules;
+
+const searchForLocations = (
+  searchText: string,
+  region: Region,
+  callback: Callback,
+  debounceMs: number = 200
+) => {
+  debounce(
+    RNReverseGeocode.searchForLocations(searchText, region, callback),
+    debounceMs
+  );
+};
+
+// This method is not available on iOS.
+// We can imply that Geocoding implementation will be always available.
+const isPresent = (
+  callback: Callback,
+  debounceMs: number = 200
+) => {
+  debounce(
+    () => callback(true),
```

Comment:
Using this `RNReverseGeocode.isPresent(console.log);` on Android worked like a charm and outputs `true` as expected; however, nothing happens on iOS. Did you manage to have it working on iOS too?